ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js

```diff
@@ -62,33 +62,55 @@ module('Acceptance | host-catalogs | hosts | read', function (hooks) {
     urls.hosts = `${urls.hostCatalog}/hosts`;
     urls.host = `${urls.hosts}/${instances.host.id}`;
     urls.unknownHost = `${urls.hosts}/foo`;
-    urls.newHost = `${urls.hosts}/new`;
-    // Generate resource couner
+
     authenticateSession({});
+    await visit(urls.hostCatalog);
   });
 
   test('visiting hosts', async function (assert) {
     assert.expect(2);
-    await visit(urls.hosts);
+    await click(`[href="${urls.hosts}"]`);
     await a11yAudit();
     assert.strictEqual(currentURL(), urls.hosts);
-    await visit(urls.host);
+
+    await click(`[href="${urls.host}"]`);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.host);
   });
 
-  test('cannot navigate to a role form without proper authorization', async function (assert) {
+  test('cannot navigate to a host form without proper authorization', async function (assert) {
```

Comment:
Same thing here, I feel like we should also test we can't visit `hosts` as well without proper permissions.